addons/l10n_din5008/report/din5008_report.xml

```diff
@@ -103,12 +103,9 @@
                         </tr>
                     </table>
                     <h2>
-                        <span t-if="not o and not docs"><t t-out="company.l10n_din5008_document_title"/></span>
```

Comment:
Basically you just got rid for the case where we no docs? (To be fair, I don't know exactly what docs is and when it could be the case but I want to understand the logic you had)

Reply:
> Basically you just got rid for the case where we no docs? (To be fair, I don't know exactly what docs is and when it could be the case but I want to understand the logic you had)

Hello @gawa-odoo,  I've analyzed the "l10n_de_document_title" field problem (history) from the 14.0 version. This field name changed to "l10n_din5008_document_title" in the 16.0 version. As per my analysis, this "l10n_de_document_title" field is not found in the "res. company" object but was used in the template, and its version is 14.0. Is there any logic to adding this field to this template?

This field was added in this commit. Please check it out.

https://github.com/odoo/odoo/commit/234fd4c1a4c6d11ecc329552f9e828bd6047f674#diff-9c749bc670f950400f52628bd830264c5de1a8436f93f4a2ccc9d7c623e711f7R109